bin/oref0-pump-loop.sh

```diff
@@ -339,7 +339,11 @@ function refresh_old_profile {
 
 function refresh_smb_temp_and_enact {
     # set mtime of monitor/glucose.json to the time of its most recent glucose value
-    touch -d "$(date -R -d @$(jq .[0].date/1000 monitor/glucose.json))" monitor/glucose.json
+    if grep "MDT cgm" openaps.ini; then
+      touch -d "$(date -R -d @$(jq .[0].date/1000 nightscout/glucose.json))" monitor/glucose.json
```

Comment:
I wonder if it'd be worthwhile making the monitor vs. nightscout directory a variable so we only have to do the if clause once.